src/main/resources/public/js/pages/store.js

```diff
@@ -179,8 +179,8 @@ const SearchAppsForm = createClass({
     fullTextSearchChanged: function (event) {
         this.props.updateFilter(null, "searchText", event.target.value);
     },
-    onGeoChange: function (geoArea) {
-        this.props.updateFilter(null, "geoAreaAncestorsUris", geoArea.ancestors);
+    onGeoChange: function (event, newValue) {
+        this.props.updateFilter(null, "geoAreaAncestorsUris", newValue);
```

Comment:
not sure about this ancestors thing. I have to check how it is supposed to be used